demo/src/main/java/net/minestom/demo/PlayerInit.java

```diff
@@ -187,24 +71,145 @@ public class PlayerInit {
         //     System.out.println("load end");
         // });
 
-        inventory = new Inventory(InventoryType.CHEST_1_ROW, Component.text("Test inventory"));
+        inventory = new Inventory(serverFacade.getGlobalEventHandler(), serverFacade.getServerSettings(), InventoryType.CHEST_1_ROW, Component.text("Test inventory"));
```

Comment:
`Inventory` should probably get the server references it needs from the player it is opened for

Reply:
help needed